test-data/unit/check-errorcodes.test

```diff
@@ -472,7 +476,10 @@ class A(TypedDict):
     two_commonparts: int
 
 a: A = {'one_commonpart': 1, 'two_commonparts': 2}
-a['other_commonpart'] = 3  # type: ignore[typeddict-item]
+a['other_commonpart'] = 3  # type: ignore[typeddict-unknown-key] \
+                           # N: Did you mean "one_commonpart" or "two_commonparts"? \
+                           # N: Error code "typeddict-item" not covered by "type: ignore" comment
```

Comment:
This looks wrong. The whole idea is that if an error is suppressed, the related notes should be suppressed as well.

Reply:
I've implemented your suggestion in commit 780e10d2d530a938246661c28348bc107f7939b0

Originally, I did it on purpose as I liked the 'Did you mean' feature. It would be helpful for situations like so:

```python
# Assume we're running mypy while ignoring the typpedict-unknown-key

F = TypedDict('F', { 'foo':int })

foo: F = { 'foo': 1 }

foo['bar'] = 2

# OOPS! I made a typo
foo['fooo'] = 2
```

My rationale being that it would be quite rare for a user to both want to add extra keys AND have those keys be very similar looking.